### PR TITLE
feat: Create backend project, first step

### PR DESCRIPTION
**Pull request by ftreguer**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+node_modules
+.vscode
+yarn.lock
```
Comment (dlecan) [inline .gitignore]: Lock file shouldn't be ignored and should be commited into git repo

File: package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "sonarplanet-services",
+  "version": "1.0.0",
+  "description": "Address tracker",
+  "main": "app.js",
+  "repository": "git@github.com:spacelephant/sonarplanet-services.git",
+  "author": "spacelephant",
+  "license": "ISC",
```
Comment (dlecan) [inline package.json]: `Apache-2.0` instead
https://spdx.org/licenses/Apache-2.0.html for the license file

**Review by dlecan — CHANGES_REQUESTED** [review]
First feedbacks, I will continue this review.

File: .editorconfig [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# http://EditorConfig.org
```
Comment (dlecan) [inline .editorconfig:1]: Missing `.ts`, `.css`, `scss` ... files configuration

**Comment by dlecan** [comment]
Missing `LICENSE` file: https://spdx.org/licenses/Apache-2.0.html

**Review by dlecan — CHANGES_REQUESTED** [review]
Some feedbacks to integrate

**Comment by dlecan** [comment]
`yarn start` doesn't work:

```
sonarplanet-services (ftreguer-v0 u=)$ yarn start
yarn run v1.6.0
$ node ./dist/app.js
module.js:478
    throw err;
    ^

Error: Cannot find module '/home/dlecan/dev/spacelephant/sonarplanet-services/dist/app.js'
    at Function.Module._resolveFilename (module.js:476:15)
    at Function.Module._load (module.js:424:25)
    at Module.runMain (module.js:611:10)
    at run (bootstrap_node.js:387:7)
    at startup (bootstrap_node.js:153:9)
    at bootstrap_node.js:500:3
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

`yarn run dev` neither:

```
sonarplanet-services (ftreguer-v0 u=)$ yarn dev run
yarn run v1.6.0
$ nodemon ./dist/app.js run
[nodemon] 1.17.3
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `node ./dist/app.js run app.js`
module.js:478
    throw err;
    ^

Error: Cannot find module '/home/dlecan/dev/spacelephant/sonarplanet-services/app.js'
    at Function.Module._resolveFilename (module.js:476:15)
    at Function.Module._load (module.js:424:25)
    at Module.runMain (module.js:611:10)
    at run (bootstrap_node.js:387:7)
    at startup (bootstrap_node.js:153:9)
    at bootstrap_node.js:500:3
[nodemon] app crashed - waiting for file changes before starting...
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+```
+geth --config rinkeby.fast.toml console
+```
+
+Web socket default port: 8546
+
+
+rinkeby.fast.toml file example:
+```
+# Note: this config doesn't contain the genesis block.
```
Comment (dlecan) [inline README.md]: Is it a problem ?
cc @Nigui

File: package.json [hunk 5]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "sonarplanet-services",
+  "version": "1.0.0",
+  "description": "Address tracker",
+  "main": "app.js",
+  "repository": "git@github.com:spacelephant/sonarplanet-services.git",
+  "author": "spacelephant",
```
Comment (dlecan) [inline package.json]: `author` should be @ftreguer :-)
See https://docs.npmjs.com/files/package.json#people-fields-author-contributors for the format.

File: package.json [hunk 6]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "sonarplanet-services",
+  "version": "1.0.0",
+  "description": "Address tracker",
+  "main": "app.js",
+  "repository": "git@github.com:spacelephant/sonarplanet-services.git",
```
Comment (dlecan) [inline package.json]: Should be https URL.

**Review by dlecan — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (ftreguer) [reply README.md]: I've removed this note

**Comment by dlecan** [comment]
Handle by #2

**Pull request closed without merging**